activities/communication/blogging.md

```diff
@@ -0,0 +1,73 @@
+---
+Type: guide
+Explains: how to blog for Foundation staff
+---
+
+# How and why to blog for Foundation staff
+
+[We blog](https://blog.publiccode.net/) for a number of different reasons. To quote the communications plan, "The blog is our primary tool for talking about:
+
+* Our work: what we’re working on and what we’ve learned
+* Our position on important issues: why we’ve made the political, technological, organizational or design decisions we have
+* Our news: what we’ve achieved and what we’ll be doing next
+* Our community: interviews with and guest posts by community members"
+
+Read the communications plan for more information about our goals, audiences and messages.
+
+## Before you start
+
+1\. Since none of us work work in isolation, check in with the communications coordinator before you start writing your blogpost. This helps us:
+
+* understand your goals and audience for the blogpost
+* establish the scope of your blogpost (to avoid duplication with other planned work)
+* schedule publication of your blogpost so it fits best with other planned communications
+
+2\. [Read our style guide](https://content-guide.18f.gov/) - we use 18F's. If you have any questions, ask!
+
+3\. Make sure your information is correct on [blog/_data/staff.csv](https://github.com/publiccodenet/blog/blob/develop/_data/staff.csv).
+
+## Write your blogpost
+
+Write your draft blogpost on HackMD.io. This lets you share it for pair writing, factchecking, feedback, or proofreading.
```

Comment:
I'd like to argue against choosing a specific tool in this guide as I think it should be up to those doing the work to choose their own toolset.